app/views/organizations/staff/pets/_pet_table.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="card">
+  <!-- table -->
+  <div class="table-responsive overflow-y-hidden">
+    <table class="table mb-0 text-nowrap table-hover table-centered">
+      <thead>
+        <tr>
```

Comment:
This `<tr>` had `style="height: 70px;"` previously in the index file. It was there to fix a small issue when only one pet was displayed in the table, the settings dropdown for that pet was cut off due to the responsive table overflow being hidden. We should be able to remove the responsive class from the table and avoid using the 70px height now that we will have a breakpoint at 1200px.
```suggestion
  <div>
    <table class="table mb-0 text-nowrap table-hover table-centered">
      <thead>
        <tr>
```

Reply:
Removed responsive table overflow being hidden